Gemfile

```diff
@@ -73,6 +73,7 @@ gem 'recaptcha', require: 'recaptcha/rails', branch: 'rails3'
 gem 'rubypants', '~> 0.2.0'
 gem 'twitter', '~> 5.14.0'
 gem 'uuidtools', '~> 2.1.1'
+gem 'psych', '~> 2.0', '>= 2.0.17'
```

Comment:
Looks like I cleared the error messages from terminal, but on `rake db:create`, I was given ~25 lines of errors related to NOT having psych as a gem. Once added, the errors were resolved and allowed me to create, migrate and seed the database.